src/styles/utils/_variables.scss

```
@@ -0,0 +1,10 @@
+// header var
+$color-blue: #00ACDC;
+$color-link: #060b35;
+
+// card var
+$card-width: 200px;
+$primary-color: #060b35;
+$secondary-color: #616070;
+$third-color: #fff;
+$fourth-color: #00acdc;
```

Why did you created the two variables with the same values?
```suggestion
$color-blue: #00ACDC;
$color-link: #060b35;

// card var
$card-width: 200px;
$secondary-color: #616070;
$third-color: #fff;
```